### PR TITLE
fix(doc): update release doc typo

### PR DESCRIPTION
**Pull request by lihongyan1**

### CODE REVIEW

File: docs/design/assets/release.uml [hunk 1]
```diff
@@ -37,7 +37,7 @@ deactivate CI
 == candidate release  ==
 Upstream -> CI: start candidate release (olm-candidate.yaml)
 CI -> Publish: candidate release (olm-publish.yaml)
-CI -> Publish: run olm-publish (channels: development, candidate)
+CI -> Publish: run olm-publish (channel: candidate)
```
Comment (sthaha) [inline docs/design/assets/release.uml:40]: @lihongyan1 , the original doc is right, when 
* code merges to main -> update development channel 
* create pre-release -> update development, candidate channels
* create release -> update development, candidate, stable channels 

so .. if someone has 
* subscribed to `development` channel they get all updates (merge to main, pre-release, release)
* subscribed to `candidate` channel, then get pre-releaes and stable release
* subscribed to `stable` channel they get only stable releases. 
*

**Review by sthaha — CHANGES_REQUESTED** [review]
this isn't the workflow :(

**Review by sthaha — CHANGES_REQUESTED** [review]
this isn't the workflow :(

**Comment by lihongyan1** [comment]
> this isn't the workflow :(

thanks, will update

**Review by sthaha — APPROVED** [review]
Thanks for the doc fix, @lihongyan1 🙇

**Pull request merged**